src/lib/pay-request/typed_clients/services/ledger/client.ts

```diff
@@ -143,7 +143,7 @@ export default class Ledger extends Client {
           .map(event => ({
             ...event,
             resource_type: event.resource_type.toUpperCase() as ResourceType,
-            payment_provider: event.payment_provider.replace(/"/g, '') as PaymentProvider
+            payment_provider: event.payment_provider && event.payment_provider.replace(/"/g, '') as PaymentProvider
```

Comment:
Should there be any that come back without payment providers? I wonder what the previous code didn't have to do this manipulation

Reply:
Might be worth sense checking that the dashboard code doesn't then expect something in a format that's changed by the typed client changing the shape of data